### DISCUSSION TITLE
Need format for Cron to run on the first of the month

### DISCUSSION THREAD

**Discussion by megarlGH** [nextgenhealthcare/connect#5794] [General]
What is the format for a Mirth Connect Cron Job to run on the first of the month?

**Comment by pacmano1** [comment]
Mirth uses the Quartz cron engine.   Just grab one of the online tools that build that for you.

**Reply by pacmano1** [reply]
Verify this but

`0 0 0 1 * ? *`

**Reply by megarlGH** [reply]
Thank you. I actually had to wind up using this:

0 0 0 1 1/1 ? *

**Reply by pacmano1** [reply]
they both do the same thing.

**Reply by megarlGH** [reply]
Thank you.

**Reply by pacmano1** [reply]
Next time please through questions in the Q and A section, so we can mark stuff as answered.

**Reply by jonbartels** [reply]
https://www.freeformatter.com/cron-expression-generator-quartz.html is a good tool for generating and validating quartz-format cron expressions.
